test/utils/utils.cpp

```diff
@@ -10,6 +10,37 @@
 #include <iomanip>
 #include <iostream>
 
+bytes from_hex(std::string_view hex)
```

Comment:
Would it be an OK idea moving this to evmc so it can also be used in Hera?

Or perhaps creating yet another library? :D

Reply:
Not sure... I'd rather copy&paste sources...